apps/prairielearn/src/schemas/schemas/questionOptionsCalculation.json

```diff
@@ -3,10 +3,29 @@
   "title": "Calculation question options",
   "description": "Options for a Calculation question.",
   "type": "object",
+  "additionalProperties": true,
```

Comment:
For my own understanding: is this not the default?

Reply:
This I think is just an artifact of `json-schema-to-zod`. I specified `.passthrough()` on the Zod schema to get this behavior.